### PR TITLE
added temp link on nav bar for login

### PR DESCRIPTION
**Pull request by pflopez**
Added temp login button

Screen Shot 2023-09-01 at 13 48 53

### CODE REVIEW

**Review by runforyourlives — APPROVED** [review]

File: src/interface/src/app/top-bar/top-bar.component.html [hunk 1]
```diff
@@ -66,6 +66,10 @@
       {{ 'help_outline' }}
     </mat-icon>
   </a>
+
+  <!--  temp login button until we wrap up new nav-->
+  <a *ngIf="showLogin$ | async" routerLink="/login" class="nav-link">Login</a>
```
Comment (sachiyuan) [inline src/interface/src/app/top-bar/top-bar.component.html:71]: Did we decide to not hide this behind any of the flags?

Comment (runforyourlives) [reply src/interface/src/app/top-bar/top-bar.component.html:71]: This does need to be hidden from the login feature flag.

**Review by runforyourlives — CHANGES_REQUESTED** [review]
can you provide a screenshot with login disabled?

File: src/interface/src/app/top-bar/top-bar.component.html (hunk 1 above)
Comment (pflopez) [reply src/interface/src/app/top-bar/top-bar.component.html:71]: it is hidden by the feature flag 

```
  showLogin$ = this.authService.isLoggedIn$.pipe(
    map(
      (loggedIn) => this.featureService.isFeatureEnabled('login') && !loggedIn
    )
  );
  
  ```

**Comment by pflopez** [comment]
closing this one in favor of https://github.com/OurPlanscape/Planscape/pull/924

**Pull request closed without merging**